wordle-web/src/views/game.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+    <div class="about">
+        <v-card>
+            <v-card-title>Word App 2: Electric Boogaloo</v-card-title>
+            <v-card-text>default placeholder body text</v-card-text>
+        </v-card>
+
+        <RouterLink to="/">Home</RouterLink>
```

Comment:
I don't know if they wanted an actual button tag or just a way to get back to the homescreen, but if you want to change this to a button it would be the following:

```suggestion
        <v-btn to="/"> Home </v-btn>
```

Reply:
Either way works! However it should at least be inside the card, as it looks a little funny outside of it. :)